### PR TITLE
Feature/sidebars refactor

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
This PR adds the layers toggles to the `local scene`. 
The state conditions that were triggering the visibility and animations  of the different sidebar items where starting to get messy. Each element (entry boxes, layer toggles, country cards...) had their own conditions that where driving the animations, the absolute positioning... so a theoretical simple stuff as adding a new box to a sidebar was turning complex and error prone.
The new approach treats each sidebar as a whole and adding elements to it will lay down each element on top of each other without the need of taking care of absolute positioning or individual elements state and animations handling.
### Designs.
Check an open box [here](https://projects.invisionapp.com/d/main#/console/17484384/362411413/preview)
### Testing instructions
Test functionality on `data-globe` page on the `local` and `global` scene modes.
Take into account that mobile designs will be tackled on a next iteration.
### Feature relevant tickets 
[Layers toggles on local scene](https://www.pivotaltracker.com/story/show/172287970)
[Sidebars refactor](https://www.pivotaltracker.com/story/show/172077969)
[Accordion items](https://www.pivotaltracker.com/story/show/169534034)

### CODE REVIEW

**Comment by kfilip94** [comment]
checking 👁️ 👁️

**Comment by kfilip94** [comment]
- Going back from the local scene destroys entry cards: 
1. Go to the landscape view
2. Select any layer
3. Go back to the global view
It happens for conservation layers widget and for human pressures widget  as well
56j9z-1ckhw

**Comment by kfilip94** [comment]
- It would be cool to perform a 180-degree turn on the arrow icon when the radio buttons group is expanded:
image

- Selecting any layer from `FINE SCALE DATA` group throws an error (both local and global scenes):
s0ket-2xnah
image

- Selecting any layer from the human pressures layers on the local scene causes the infinite spinner animation, it doesn't disappear after the layer load or even after refresh.
image

**Comment by weberjavi** [comment]
@kfilip94 I would not bother to change any of the Human layers related code (I'm talking about the loading spinner now) since all of this code is updated or removed on the current `develop` branch

File: src/components/human-impact-sidebar-card/human-impact-sidebar-card-component.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { LAYERS_CATEGORIES } from 'constants/mol-layers-configs';
+import CategoryBox from 'components/category-box';
+import HumanImpactLayers from 'components/human-impact-layers';
+import styles from './human-impact-sidebar-card-styles.module.scss'
+
+const humanImpact = LAYERS_CATEGORIES.LAND_PRESSURES;
+
+const BiodiversitySidebarCardComponent = ({activeLayers, handleGlobeUpdating, countedActiveLayers, setRasters, rasters, map, view}) => {
```
Comment (kfilip94) [inline src/components/human-impact-sidebar-card/human-impact-sidebar-card-component.jsx:10]:
```suggestion
const HumanImpactSidebarCardComponent= ({activeLayers, handleGlobeUpdating, countedActiveLayers, setRasters, rasters, map, view}) => {
```

File: src/components/human-impact-sidebar-card/human-impact-sidebar-card-component.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            setRasters={setRasters}
+            map={map}
+            view={view}
+          />
+      </div>
+    </div>
+  )
+}
+
+export default BiodiversitySidebarCardComponent;
```
Comment (kfilip94) [inline src/components/human-impact-sidebar-card/human-impact-sidebar-card-component.jsx:37]:
```suggestion
export default HumanImpactSidebarCardComponent;
```

File: src/components/mask-country-manager/mask-country-manager.js [hunk 3]
```diff
@@ -67,10 +68,10 @@ const MaskCountryManager = props => {
   }, [countryExtent, isCountryMode]);
 
   useEffect(() => {
-    if (graphicsLayer && !countryISO) {
+    if (graphicsLayer && (!countryISO || sceneMode === GLOBAL_SCENE)) {
```
Comment (kfilip94) [inline src/components/mask-country-manager/mask-country-manager.js:71]: You can use `isCountryMode` prop, it's already there

File: src/components/protected-areas-sidebar-card/protected-areas-sidebar-card-component.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { LAYERS_CATEGORIES } from 'constants/mol-layers-configs';
+import CategoryBox from 'components/category-box';
+import ProtectedAreasLayers from 'components/protected-areas-layers';
+import styles from './protected-areas-sidebar-card-styles.module.scss'
+
+const protectedAreas = LAYERS_CATEGORIES.PROTECTION;
+
+const BiodiversitySidebarCardComponent = ({activeLayers, activeCategory, handleGlobeUpdating, countedActiveLayers,map}) => {
```
Comment (kfilip94) [inline src/components/protected-areas-sidebar-card/protected-areas-sidebar-card-component.jsx:10]:
```suggestion
const ProtectedAreasSidebarCardComponent = ({activeLayers, activeCategory, handleGlobeUpdating, countedActiveLayers,map}) => {
```

File: src/components/protected-areas-sidebar-card/protected-areas-sidebar-card-component.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          handleGlobeUpdating={handleGlobeUpdating}
+          activeLayers={activeLayers}
+          activeCategory={activeCategory}
+        />
+      </div>
+    </div>
+  )
+}
+
+export default BiodiversitySidebarCardComponent;
```
Comment (kfilip94) [inline src/components/protected-areas-sidebar-card/protected-areas-sidebar-card-component.jsx:34]:
```suggestion
export default ProtectedAreasSidebarCardComponent;
```

File: src/constants/layers-slugs.js [hunk 6]
```diff
@@ -11,6 +11,7 @@ export const GRID_CELLS_FOCAL_SPECIES_FEATURE_LAYER = 'grid_cells_focal_species'
 export const BIODIVERSITY_FACETS_LAYER = 'biodiversity_facets';
 // Dinamically created graphics layer to paint aggregated gridcells
 export const GRAPHIC_LAYER = 'graphic_layer';
+export const COUNTRY_MASK_LAYER = 'mask-layer';
```
Comment (kfilip94) [inline src/constants/layers-slugs.js:14]: Thanks for creating constant for the mask layer! 👍

**Comment by kfilip94** [comment]
Seem that after last commits I can expand all the category boxes in the same time, I assume it's not intended?
image

File: src/components/mask-country-manager/mask-country-manager.js (hunk 3 above)
Comment (weberjavi) [reply src/components/mask-country-manager/mask-country-manager.js:71]: 🤦

**Comment by weberjavi** [comment]
Yeah it is intended. I've moved the logic from the store to component.

**Pull request merged**